web/modules/custom/joinup_core/joinup_core.module

```diff
@@ -490,6 +490,27 @@ function joinup_core_entity_create(EntityInterface $entity) {
   }
 }
 
+/**
+ * Implements hook_entity_insert().
+ */
+function joinup_core_entity_insert(EntityInterface $entity) {
+  // List the related entity bundles.
+  $related_bundles = [
+    'collection',
+    'solution',
+  ];
+
+  // Check to the current bundle is in our list.
+  if (in_array($entity->bundle(), $related_bundles)) {
```

Comment:
Because you've implemented `hook_entity_insert()` this check is not consistent check. It should:

* Either, check also if `$entity->getEntityTypeId() === 'rdf_entity'`
* Or, use `hook_ENTITY_TYPE_insert()`, in this case `joinup_core_rdf_entity_insert()`,

Of course, we don't have a node-type named `collection` but we have be safe.

Reply:
Since this logic is placed in joinup_core_form_rdf_entity_form_alter() I don't think we really need to check the EntityTypeID because this function is called only when it's an rdf entity itself.